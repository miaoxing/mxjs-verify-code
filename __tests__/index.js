import React from 'react';
import {render} from '@testing-library/react';
import {MobileVerifyCode} from '..';
import {Form} from '@mxjs/form';
import {MemoryRouter} from 'react-router';

describe('test', () => {
  test('basic', () => {
    const result = render(<MemoryRouter>
      <Form>
        <MobileVerifyCode/>
      </Form>
    </MemoryRouter>);
    expect(result.container).toMatchSnapshot();
  });
});
