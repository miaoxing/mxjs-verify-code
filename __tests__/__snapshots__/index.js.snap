// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test basic 1`] = `
<div>
  <form>
    <div
      class="js-verify-code-from-group form-group"
    >
      <label
        class="control-label"
        for="verify-code"
      >
        验证码
         
        <span
          class="text-warning"
        >
          *
        </span>
      </label>
      <div
        class="col-control"
      >
        <div
          class="input-group"
        >
          <input
            class="form-control"
            id="verify-code"
            maxlength="6"
            name="verifyCode"
            placeholder="请输入验证码"
            type="tel"
            value=""
          />
          <span
            class="input-group-append"
          >
            <button
              class="js-verify-code-send btn btn-outline-primary"
              type="button"
            >
              获取验证码
            </button>
          </span>
        </div>
      </div>
    </div>
  </form>
</div>
`;
